workers/unity/Packages/com.improbable.gdk.core/Logging/ILogDispatcher.cs

```diff
@@ -4,9 +4,20 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     The ILogDispatcher interfaces is used to implement different types of loggers. By default, the
+    ///     ILogDispatcher supports structured logging.
```

Comment:
not true, structured logging is directly implemented in the ForwardingDispatcher

Reply:
This was mostly in reference to the interface requiring you to implement `HandleLog(LogType type, LogEvent logEvent)` where `LogEvent` encapsulates the structured logging data - the interface supports structured logging from that POV

What you do with the data is entirely up to you - might be too pedantic though

Reply:
discussed offline, we'll leave it like this